Solution1/WeatherApi/appsettings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    },
+    "Console": {
+      "LogLevel": {
+        "Default": "Information",
```

Comment:
full duplicate of Logging, it's not necessary, it will be inherited from Logging

Reply:
+